test/verify/check-service

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/python3
+
+# checkpoint:
+# 1. osbuild-composer.socket can be started and enabled
+# 2. no blueprint by default and can add blueprint
+
+import os
+import json
+import string
+import random
+from hashlib import sha256
+from pathlib import Path
+
+import parent
+from testlib import *
+
+
+@nondestructive
```

Comment:
You already made those tests nondestructive? Nice!!

Reply:
Yes.